dnf-plugins-extras.spec

```diff
@@ -32,6 +32,7 @@ Extras Plugins for DNF.
 
 %package -n %{name}-common-data
 Summary:        Common data files for Extras Plugins for DNF
+Provides:       %{name} = %{version}-%{release}
```

Comment:
yeah, to some degree. probably we should revert with requirements ONLY for packages which do not require anything else (I think candidates are repomanage and debug)....